docs/install_guides/installing-openvino-pip.md

```diff
@@ -1,17 +1,40 @@
+
+
+## Introduction
+
+OpenVINO™ toolkit is a comprehensive toolkit for quickly developing applications and solutions that solve a variety of tasks including emulation of human vision, automatic speech recognition, natural language processing, recommendation systems, and many others. Based on latest generations of artificial neural networks, including Convolutional Neural Networks (CNNs), recurrent and attention-based networks, the toolkit extends computer vision and non-vision workloads across Intel® hardware, maximizing performance. It accelerates applications with high-performance, AI and deep learning inference deployed from edge to cloud.
+
+The Intel® Distribution of OpenVINO™ toolkit for Linux\*:
+- Enables CNN-based deep learning inference on the edge
+- Supports heterogeneous execution across Intel® CPU, Intel® Integrated Graphics, Intel® Neural Compute Stick 2, and Intel® Vision Accelerator Design with Intel® Movidius™ VPUs
+- Speeds time-to-market via an easy-to-use library of computer vision functions and pre-optimized kernels
+
+**Included with the Installation and installed by default:**
+
+| Component                                                                                           | Description                                                                                                                                                                                                                                                                                                   |  
+|-----------------------------------------------------------------------------------------------------|---------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------|
+| [Inference Engine](../IE_DG/inference_engine_intro.md)               | This is the engine that runs the deep learning model. It includes a set of libraries for an easy inference integration into your applications.                                                                                                                                                                |
 
 ## System Requirements
 
-* [Python* distribution](https://www.python.org/) 3.6 or 3.7
+* [Python* distribution](https://www.python.org/) 3.6, 3.7, 3.8
 * Operating Systems:
-  - Ubuntu* 18.04 long-term support (LTS), 64-bit
+  - Ubuntu* 18.04 long-term support (LTS), 64-bit (python 3.6 or 3.7)
+  - Ubuntu* 20.04 long-term support (LTS), 64-bit (python 3.8)
   - macOS* 10.15.x versions
   - Windows 10*, 64-bit Pro, Enterprise or Education (1607 Anniversary Update, Build 14393 or higher) editions
   - Windows Server* 2016 or higher
 
```

Comment:
Add this: "**NOTE:** This package can be installed on many versions of Linux, but only the specific versions above are fully validated."

Reply:
Added

Reply:
Added